cli/src/commands/rebase.rs

```diff
@@ -143,6 +143,12 @@ pub(crate) struct RebaseArgs {
     /// commit)
     #[arg(long, short, required = true)]
     destination: Vec<RevisionArg>,
+
+    /// If true, when rebasing would produce an empty commit, the commit is
+    /// skipped.
```

Comment:
Mention here (and probably on the enum variant) that merge commits are not skipped.